server/server.js

```diff
@@ -42,6 +44,38 @@ process.env['NODE_ENV']='production'
 // place holder for the data
 const users = [];
 
+app.use('/ws/', createProxyMiddleware({
+  target: cfg.serverConfig.WEBSOCKET_URL,
+  'option.ws': true,
+}));
+
+app.use('/api/v1/standalone/functions', createProxyMiddleware({
```

Comment:
The name of the cluster, ie `standalone`, needs to be passed in by configuration. From what I can tell, this code will only work with a development Pulsar instance. For example, in Kubernetes, the default cluster name is `pulsar` and the endpoints need to be in the form `/api/v1/pulsar/functions` and the rewrite needs to strip out `pulsar` and change the endpoint to `/admin/v3` (in the case of functions).

Reply:
There are 2 places in the `dashboard/public/config.js` file to configure the cluster name:
    "home_cluster": "standalone",
    "cluster_list": ["standalone"],
There is code to support multiple clusters, but it is not fully exposed. So this is just a list of 1 cluster. The "home_cluster" is the one it starts at on the list.